Sources/NetworkProtection/Storage/NetworkProtectionSelectedServerStore.swift

```diff
@@ -17,25 +17,13 @@
 //
 
 import Foundation
-
+/*
 protocol NetworkProtectionSelectedServerStore: AnyObject {
```

Comment:
One note is that we'll have to add something like this back to handle geoswitching, I believe